src/main/java/dumshenko/daniil/Square.java

```diff
@@ -0,0 +1,25 @@
+package dumshenko.daniil;
+
+public class Square extends Rectangle {
+
+    private double squareWidth;
+    private double squareHeight;
+
+    public Square(FigureType typeOfFigure, double squareWidth, double squareHeight) {
+        super(typeOfFigure, squareWidth, squareHeight);
+        this.squareWidth = squareWidth;
+        this.squareHeight = squareHeight;
+    }
+
+    @Override
+    public void displayInfo() {
+        System.out.println("I`m Square ");
+        System.out.println("My width: " + squareWidth);
+        System.out.println("My height: " + squareHeight);
+    }
+
+    @Override
+    public double calculateArea() {
+        return super.calculateArea();
+    }
```

Comment:
nit: In this case the override does not do anything useful since we just directly call super with the same method. Burn this with fire 😄 
```suggestion
```